webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -107,7 +107,7 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
 export default TrialInfoBox;
 
 export const TrialInfoBoxMultiTrial: React.FC<Props> = ({ experiment }: Props) => {
-  const { searcher } = experiment?.configRaw;
+  const { searcher } = experiment?.configRaw ?? { searcher: undefined };
```

Comment:
Could this be better as `const searcher = experiment?.configRaw?.searcher` ?

Reply:
i think we can. actually we dont even need optional chaining